### ISSUE TITLE
Adding new sections to the documentation

### ISSUE THREAD

**Issue by carodak** [DouglasNeuroInformatics/Douglas_User_Documentation#44]
Hi,

Thank you for this documentation. I would say that a complete usage example would be very useful afterwards. As well as maybe a "How to contribute" category to a contact link if anyone is interested in contributing. A Prerequisite category and possibly a link to the Reference API might also be useful.

Personally, I also like to have a core features section that allows me to quickly grasp what I can do with the platform.

Best,
Caroline

**Comment by gdevenyi** [comment]
> I would say that a complete usage example would be very useful afterwards.

I'm not sure what you're asking for here.

> As well as maybe a "How to contribute" category to a contact link if anyone is interested in contributing.

We can add that, the contact method will remain Github

> A Prerequisite category and possibly a link to the Reference API might also be useful.

I have again no idea what you're referring to here, this is reference documentation for a computer system.

**Comment by carodak** [comment]
Indeed, I mixed my remarks with my old guidelines without deleting the parts that did not apply here. Sorry for the confusion, I have edited my post (+representativeness of the data, +ressource to avoid lab errors).

**Comment by surchs** [comment]
Thanks @carodak for these ideas and suggestions! 

I see a number of things here that would be easier to track in separate issues.
1. Add a specific "Howto contribute" section. I think this is a great idea and also very important for our community building efforts. I can see that Gabe has added #45 for this
2. I like the idea of a "core features" overview at the top level. @carodak, could you open a separate issue for this where you explain what you have in mind (with a simple example) and reference this issue here
3. > The only thing I would change would be to add the emphasis on the representativeness of the data. It is important to ensure that the data are representative of the study population.

Thanks for this suggestion. Again, a separate issue would be helpful here and allow you to give additional context to your idea. e.g. what aspects of representativeness would you like to see discussed here and what resources could you suggest. This could also be a great opportunity for you to contribute this section directly to our documentation. Let me know if you'd like to have some guidance with that.
4. > I don't know where this would fit but it is also an interesting resource that has been shared in my lab: https://psyarxiv.com/rsn5y (avoiding data collection errors, storing, standardization, data analysis..)

Thanks for the link to the article! I am also not sure where this would fit. But it would be nice to have a repository of articles and resources for research related topics that we may still add to the docs.

**Comment by surchs** [comment]
@all-contributors, please add @carodak for ideas
